com.gitlab.davem.ClamTk.yaml

```diff
@@ -0,0 +1,121 @@
+app-id: com.gitlab.davem.ClamTk
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```

Comment:
Have you tried building it using 19.08?

Reply:
Yes please, while 18.08 is still supported, we definitely don't want more apps on "old" runtimes.